### PR TITLE
Projectorsize selection with slider

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/core/static/templates/core/manage-projectors.html [hunk 1]
```diff
@@ -82,19 +83,30 @@ <h1 translate>Manage projectors</h1>
...
+                    <a href ng-click="setAspectRatio(projector, aspectRatio); saveResolution(projector);">
+                      <i class="fa fa-check" ng-if="aspectRatio === resolutions[projector.id].aspectRatio"></i>
+                      {{ aspectRatio }}
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div>
+              <rzslider
```
Comment (emanuelschuetze) [inline openslides/core/static/templates/core/manage-projectors.html]: suggestion/wish: add a custom css class to use our nicer OpenSlides corporate blue for slider color.

File: openslides/core/static/js/core/site.js [hunk 2]
```diff
@@ -1428,6 +1429,21 @@ angular.module('OpenSlidesApp.core.site', [
...
         ProjectionDefault.bindAll({}, $scope, 'projectiondefaults');
 
+        /* Info on resolution calculating:
+         * Internally the resolution is saved as (width, height) but the user has
+         * an aspect ratio to choose and a width from 800 to 3840 (4K).*/
+        $scope.aspectRatios = {
+            '4:3': 4/3,
+            '16:9': 16/9,
+            '16:10': 16/10,
+            '21:9': 21/9,
```
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/site.js]: Done.

File: bower.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "angular-ui-router": "~0.3.1",
     "angular-ui-tree": "~2.22.0",
     "angular-xeditable": "~0.5.0",
+    "angularjs-slider": "~6.2.2",
```
Comment (FinnStutzenstein) [inline bower.json]: Done.

File: openslides/core/static/templates/core/manage-projectors.html (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/core/static/templates/core/manage-projectors.html]: Done.

**Pull request merged**